### ISSUE TITLE
Puppet lint syntax error - puppet parser validate no issues

### ISSUE THREAD

**Issue by cdenneen** [rodjek/puppet-lint#746]
```
site/role/manifests/web.pp:8:syntax:ERROR:Syntax error 
(try running `puppet parser validate <file>`)

  include ::monitoring_scripts
           ^
```
```puppet
#description
class role::web{
  include ::profile::web
  include ::profile_base::windows
  include ::profile::scheduledtask
  include ::profile::regedit
  #lint:ignore:roles_and_profiles
  include ::monitoring_scripts
  #lint:endignore
}
```
```
  * puppet-lint (2.3.0)
  * puppet-lint-absolute_classname-check (0.2.4)
  * puppet-lint-classes_and_types_beginning_with_digits-check (0.1.2)
  * puppet-lint-empty_string-check (0.2.2)
  * puppet-lint-leading_zero-check (0.1.1)
  * puppet-lint-trailing_comma-check (0.3.2)
  * puppet-lint-undef_in_function-check (0.2.1)
  * puppet-lint-unquoted_string-check (0.3.0)
  * puppet-lint-variable_contains_upcase (1.2.0)
  * puppet-lint-version_comparison-check (0.2.1)
```
```
>> be puppet parser validate site/role/manifests/web.pp --debug --trace --verbose
Debug: Runtime environment: puppet_version=4.10.1, ruby_version=2.1.8, run_mode=user, default_encoding=UTF-8
```

**Comment by cdenneen** [comment]
Looks like this is fixed in master... can we get a release? @rodjek think you fixed this first commit after the 2.3.0 release...

**Comment by cdenneen** [comment]
Looks like fixed in c6fc417ce09b7ca9f3d6a15e1ffce28a1abfb096

**Comment by cdenneen** [comment]
Closing looks like some non-printing character is causing the issue... but it was fixed with that commit... not sure why that patch fixes the non-printing character issue
